practical_05/word_occurrences.py

```diff
@@ -0,0 +1,20 @@
+text = input("Text: ")
+word_to_count = {}
+words = text.split(" ")
+for word in words:
+    if word in word_to_count:
+        word_to_count[word] += 1
+    else:
+        word_to_count[word] = 1
+strings = []
+longest_word = words[0]
```

Comment:
Is calculating the lengths of words and storing the longest word 'str' and then recalculating the length 'int' of that word the most efficient? Why not store the length 'int' which is what your trying to find?

Reply:
Wouldn't storing the length of the word not change anything except creating an extra variable? If I added length_of_longest_word = len(longest_word) at the end of a for loop it would still the same amount of calculation as having it in the append statement